### ISSUE TITLE
Built-in http server image cache issue

### ISSUE THREAD

**Issue by bekatd** [r0x0r/pywebview#604]
### Specification
- pywebview version: 3.3.3
- platform / version: Windows 10

### Description  
  
I am trying to serve image from the same location.
The case is that I am rewriting image in-place and image name remains same.
problem is that built-in http-server caches first image and then never updates it with new image content.
I thought changing image source to another image and then back to the same image would force http-server to reload image again, but no call is made in the networks tab and its clear that http server uses cached image.
I have not found anything in docs to force http server to load image again whenever image source is changed.
In other words, http-server MUST make a request every time image(or any other resource) source is changed. Would be great to have option for htttp-server to enable/disable cache.
  
### Practicalities

- NO I am willing to work on this issue myself.

- NO I am prepared to support this issue financially.

**Comment by r0x0r** [comment]
If no network call is made, does not that mean that web renderer is responsible for caching?

**Comment by bekatd** [comment]
> If no network call is made, does not that mean that web renderer is responsible for caching?

I am not sure. I am just updating image source path back and forth after changing image itself, to retrigger request. In the networks tab I see that image is served by some local port/server but no request is made after first load

**Comment by r0x0r** [comment]
You can check if a correct image is served by opening the image URL using a
browser.
